core/src/main/java/feign/RequestTemplate.java

```diff
@@ -593,7 +609,7 @@ public String toString() {
    * Replaces query values which are templated with corresponding values from the {@code unencoded}
    * map. Any unresolved queries are removed.
    */
-  public void replaceQueryValues(Map<String, ?> unencoded) {
```

Comment:
please overload with the prior signature so that we don't break other people's code.